support-frontend/assets/pages/supporter-plus-landing/components/threeTierCard.tsx

```diff
@@ -250,9 +252,24 @@ export function ThreeTierCard({
 							}
 						})}
 					</span>
-					<span css={benefitsPrefixPlus}>plus</span>
 				</div>
 			)}
+			{offers?.description && (
+				<div css={benefitsPrefixCss}>
+					<span>
+						{offers.description.map((stringPart) => {
+							if (typeof stringPart === 'string') {
+								return stringPart;
+							} else {
+								return <strong>{stringPart.copy}</strong>;
+							}
+						})}
+					</span>
+				</div>
+			)}
+			{(benefits.description || offers?.description) && (
```

Comment:
We want this to show if you have the `benefits.description`? Isn't it just `offers` specific?

Reply:
This applies to the `--- plus ---` line. Wouldnt you want it if either the benefits description or the offers description (or both) were displayed?